Source/engine/assets.cpp

```diff
@@ -149,11 +152,13 @@ AssetRef FindAsset(std::string_view filename)
 
 	// Files in the `PrefPath()` directory can override MPQ contents.
 	{
-		const std::string path = paths::PrefPath() + relativePath;
-		result.directHandle = OpenOptionalRWops(path);
-		if (result.directHandle != nullptr) {
-			LogVerbose("Loaded MPQ file override: {}", path);
-			return result;
+		for (const auto &overridePath : OverridePaths) {
+			const std::string path = overridePath + relativePath;
+			result.directHandle = OpenOptionalRWops(path);
+			if (result.directHandle != nullptr) {
+				LogVerbose("Loaded MPQ file override: {}", path);
+				return result;
+			}
 		}

```

Comment:
@glebm if you can add the equivalent to the UNPACKED_MPQS portion mods should work there as well. In some of the previous PRs I also tired to make more things work when things are build like by testing for expected files.

Reply:
Hopefully this should work as is with UNPACKED_MPQS already, if not let's fix it in a follow-up PR

Reply:
Ah, I think to make it work with `UNPACKED_MPQS` we simply need to adjust `FindUnpackedMpqFile` to look in the mods directories taking priorities into account.